pkg/splunk/enterprise/types.go

```diff
@@ -56,6 +56,9 @@ const (
 
 	// SplunkMonitoringConsole is a single instance of Splunk monitor for mc
 	SplunkMonitoringConsole InstanceType = "monitoring-console"
+
+	// TmpAppDownloadDir is the Operator directory for app framework, when there is no explicit volume specified
+	TmpAppDownloadDir string = "/tmp/appframework/"
```

Comment:
@sgontla Would it make sense to apply this [suggestion](https://github.com/splunk/splunk-operator/pull/671#discussion_r800919312) in this case of Operator too? Something like "/tmp/operator-staging/appframework/"
cc: @gaurav-splunk

Reply:
@smohan-splunk , for the operator, in the case of volume mount, we are already having that kind of directory hierarchy, i.e `/opt/splunk/appframework/`. We may not need the keyword `operator`, as it is already on the operator?